package.json

```diff
@@ -27,5 +27,9 @@
     "babel": "^5.6.23",
     "lodash": "^3.10.0",
     "tape": "^4.0.0"
```

Comment:
Sure. Consider this entire repo not as a real module or something production ready.